frontend/src/components/Forms/AddPermitForm.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import PropTypes from "prop-types";
+import { Field, reduxForm, formValueSelector } from "redux-form";
+import RenderField from "@/components/common/RenderField";
+import RenderSelect from "@/components/common/RenderSelect";
+import RenderDate from "@/components/common/RenderDate";
+import { Form, Button, Col, Row, Popconfirm, Tabs } from "antd";
+import * as FORM from "@/constants/forms";
+import { required, dateNotInFuture } from "@/utils/Validate";
+import { resetForm } from "@/utils/helpers";
+import { getDropdownPermitStatusOptions } from "@/selectors/staticContentSelectors";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  permitStatusOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  title: PropTypes.string.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  permitTypeCode: PropTypes.string,
+  permitActivityTypeCode: PropTypes.string,
+};
+
+const defaultProps = {
+  permitTypeCode: "",
+  permitActivityTypeCode: "",
+};
+
+const permitTypes = [
+  {
+    label: "Coal",
+    value: "C",
+  },
+  {
+    label: "Mineral",
+    value: "M",
+  },
+  {
+    label: "Placer",
+    value: "P",
+  },
+  {
+    label: "Sand & Gravel",
+    value: "G",
+  },
+  {
+    label: "Quarry",
+    value: "Q",
+  },
+];
```

Comment:
It looks like this should live on the server at an options endpoint

Reply:
This dropdown generates a prefix for the permit number, which is then passed to the backend. It only exists to enforce from formatting on the input. It 'could' be on the backend, but wouldn't have any FK or data integrity/quality improvements.  Same for the permit activity type below.

Reply:
Minus, the `Quarry` these are all the same as tenure_types that we already fetch from the backend, and have various selectors for (including dropdown format and hash format)

Two questions, are permits based on the tenures? If so, should we update that table to include `Quarry` and use the tenur_type options in this situation?

Reply:
I guess my question is where did these options come from? Did they require business knowledge that isn't available in our API? Or was it a case of the frontend wanting to provide some options to the end user that has no business value?

If it's the latter then I think it's fine to leave it in the frontend. It just **looks** like something that I'd expect to see in a backend code table

Reply:
These are specifically for data-entry nicety and the result is a string permit_no "CX-1234". The list was given to us, and although we could have had a table and API for this we thought the value wasn't there. As this explicit type does get discarded, I don't see a case where this would be used elsewhere.

I didn't know they were almost exactly the same list as tenure types. If permits are based on tenures, it would totally make sense to use that as the source and I would have pushed for that at the start. Let's see if we can confirm that.

If we have to start adding flags or other meta to identify which should and shouldn't appear on this drop down or that dropdown, I think this will be contained enough that the hard-coded list here makes more sense.